controllers/application_controller.go

```diff
@@ -126,6 +127,9 @@ func (r *ApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 					// if fail to delete the external dependency here, log the error, but don't return error
 					// Don't want to get stuck in a cycle of repeatedly trying to delete the repository and failing
 					log.Error(err, "Unable to delete GitOps repository for application %v in namespace %v", application.GetName(), application.GetNamespace())
+
+					// Increment the Application deletion failed metric as the application delete did not fully succeed
+					metrics.ApplicationDeletionFailed.Inc()
```

Comment:
Are we going to count the [parse error](https://github.com/redhat-appstudio/application-service/blob/main/controllers/application_controller_finalizer.go#L56) as part of the SLO?  Will the finalizer get called if the Application CR was not created properly due to bad input?

Reply:
I’ll update to count the parse error.

Bad input won’t prevent the finalizer from executing

Reply:
right, which means we shouldn't be counting the parse error or the error if the gitops repo URL attribute is not found?